Analyzer/plugins/Analyzer.cc

```diff
@@ -521,6 +523,33 @@ void Analyzer::analyze(const edm::Event& iEvent, const edm::EventSetup& iSetup)
   //===================== Handle For PFCandidate ===================
   const edm::Handle<reco::PFCandidateCollection> pfCandHandle = iEvent.getHandle(pfCandToken_);
 
+  
+  //===================== Handle For Muons ===================
+  unsigned int Muons_count = 0; 
+  float maxPtMuon1 = 0, maxPtMuon2 = 0;
+  float etaMuon1 = 0, phiMuon1 = 0;
+  float etaMuon2 = 0, phiMuon2 = 0;
+  unsigned int muon1 = 0;
+  vector<reco::Muon> muonCollHandle = iEvent.get(muonToken_);
```

Comment:
This is not actually the handle, but the collection itself
(you get the handle by using `iEvent.getHandle()`
I mean this works technically, just misleading naming